File: plop-templates/adapter/acquiring/src/converter/request/EntryStateToPreAuthRequestConverter.java
package com.rbkmoney.{{adapterAcquiringPackageCase bank_name}}.converter.request;

import com.rbkmoney.{{adapterAcquiringPackageCase bank_name}}.client.model.PreAuthRequest;
import com.rbkmoney.{{adapterAcquiringPackageCase bank_name}}.model.CustomEntryStateModel;
import org.springframework.core.convert.converter.Converter;
import org.springframework.stereotype.Component;

@Component
public class EntryStateToPreAuthRequestConverter implements Converter<CustomEntryStateModel, PreAuthRequest> {

    @Override
    public PreAuthRequest convert(CustomEntryStateModel entryStateModel) {
        return new PreAuthRequest();
    }
}

File: plop-templates/adapter/acquiring/src/converter/request/EntryStateToPreAuthRequestConverter.kt
package com.rbkmoney.{{adapterAcquiringPackageCase bank_name}}.converter.request

import com.rbkmoney.{{adapterAcquiringPackageCase bank_name}}.client.model.PreAuthRequest
import com.rbkmoney.{{adapterAcquiringPackageCase bank_name}}.model.CustomEntryStateModel
import org.springframework.core.convert.converter.Converter
import org.springframework.stereotype.Component

@Component
class EntryStateToPreAuthRequestConverter : Converter<CustomEntryStateModel, PreAuthRequest> {

    override fun convert(entryStateModel: CustomEntryStateModel): PreAuthRequest {
        return PreAuthRequest()
    }
}
